lib/common.sh

```diff
@@ -103,12 +103,15 @@ fi
 
 export M3PATH="${M3PATH:-${GOPATH}/src/github.com/metal3-io}"
 export BMOPATH="${BMOPATH:-${M3PATH}/baremetal-operator}"
+export BAREMETAL_OPERATOR_LOCAL_IMAGE="${BMOPATH}"
 # shellcheck disable=SC2034
 export RUN_LOCAL_IRONIC_SCRIPT="${BMOPATH}/tools/run_local_ironic.sh"
 
 export CAPM3PATH="${CAPM3PATH:-${M3PATH}/cluster-api-provider-metal3}"
 export CAPM3_BASE_URL="${CAPM3_BASE_URL:-metal3-io/cluster-api-provider-metal3}"
-export CAPM3REPO="${CAPM3REPO:-https://github.com/${CAPM3_BASE_URL}}"
+export CAPM3REPO="${CAPM3REPO:-https://github.com/ardaguclu/cluster-api-provider-metal3}"
+export CAPM3BRANCH="inject-bmo-ca-from"
```

Comment:
would be better to specify this in line 129, so CI will use this branch in v1a4 tests, and master in v1a5